components/admin_console/manage_roles_modal/manage_roles_modal.tsx

```diff
@@ -14,8 +13,27 @@ import {trackEvent} from 'actions/diagnostics_actions.jsx';
 import FormattedMarkdownMessage from 'components/formatted_markdown_message.jsx';
 import BotBadge from 'components/widgets/badges/bot_badge';
 import Avatar from 'components/widgets/users/avatar';
+type Props = {
+    show: boolean;
+    user?: any;
```

Comment:
Quick question:  is the intention to make `user` a specific type when more code gets ported to TS?

Reply:
thats a good question but i dont know about a plan to do so, imo its a good idea to do so if there's any "starter" type for the `user` object i can change to that type and we can append to it as we go.